sphinx/workshop/reference.rst

```diff
@@ -14,6 +14,70 @@ Part 2
 
 You can download the helper library :download:`here <../../lib/basic_python3.py>`.
 
+Lists
+^^^^^^^^^^^^^^^^^^^^^
+
+Lists are structures that contain multiple values, and can contain numbers, strings and even other lists. 
+Like strings, you can add lists together, and assign them to variables. 
+You can also access an element of a list by using the *index* of the element. The first element has index 0. 
+
+.. code-block:: python
+
+    >>> lst = [1,2,3]
+    >>> lst2 = ["a","b","c"]
+    >>> lst + lst2
+    [1, 2, 3, 'a', 'b', 'c']
+    >>> newlst = lst + lst2
+    >>> newlst
+    [1, 2, 3, 'a', 'b', 'c']
+    >>> newlst[0]
+    1
+
+For loops
+^^^^^^^^^^^^^^^^^^^^^
+
+For loops enable us to loop over the elements in a list or string. 
+
+.. code-block:: python
+
+    >>> lst = [1,2,3,4]
+    >>> x = 0
+    >>> for i in lst:
+    	x = x + i
+    >>> x
+    10
+    >>> newlist = []
+    >>> for i in lst:
+        newlist = [i] + newlist
+    >>> newlist
+    [4,3,2,1]
+
+Conditionals (if, elif, else)
+^^^^^^^^^^^^^^^^^^^^^
+
+Using if, elif and else, we can impose conditions to control what our code does. "Elif" is short for "else if".
```

Comment:
Relic of past commits here. If you're on a *nix shell, you can use

```
$ grep elif referene.rst
```

To print out all the lines that have the string `elif` in them, in the file `reference.rst`.